src/settingsui.ts

```diff
@@ -519,10 +579,12 @@ export function renderSettings() {
 				{ desc: `Go to the Crankshaft <a href="https://github.com/${repoID}#userscripts">README.md</a> to download some made by the client dev.` })));
 		}
 
+		// This array is used to store userscript settings HTML
+		var customUserScriptSettings: Array<DocumentFragment> = []
```

Comment:
why `var`? wouldn't `let` suffice?
```suggestion
		var customUserScriptSettings: DocumentFragment[] = []
```